scripts/standardise_validate.py

```diff
@@ -47,7 +47,7 @@ def main() -> None:
             continue
 
         # Validate the file
-        file_check = FileCheck(file, scale, srs)
+        file_check = FileCheck(file, int(arguments.scale), srs)
         if not file_check.validate():
             vfs_path = ""
             env_argo_template = os.environ.get("ARGO_TEMPLATE")
```

Comment:
There is an [`is_argo()` function](https://github.com/linz/topo-imagery/blob/47e966a675f40a9da578fbbabb612f9dc0e672f1/scripts/cli/cli_helper.py#L38).
Do we need to log this if it's run in a container outside of Argo?

Reply:
We mostly run it outside Argo for testing, so I did not want to duplicate the log line in that case. Instead it will log an empty path

Reply:
`is_argo()` returns a boolean, using it will call `os.environ.get()` twice in this case

Reply:
Can you use it on line 54 instead? And remove this check?